3scaleAdapter/pkg/threescale/threescale.go

```diff
@@ -50,6 +50,114 @@ type (
 // For this PoC I'm using the authorize template, but we should check if the quota template
 // is more convenient and we can do some optimizations around.
 var _ authorization.HandleAuthorizationServiceServer = &Threescale{}
+var _ logentry.HandleLogEntryServiceServer = &Threescale{}
+
+func (s *Threescale) HandleLogEntry(ctx context.Context, l *logentry.HandleLogEntryRequest) (*v1beta1.ReportResult, error) {
```

Comment:
Are we missing a test case for this @jmprusi?

Reply:
Yes, working on them now! WIP ;)

Reply:
@PhilipGough check now please!